### ISSUE TITLE
Temporal artery pulses reduced or absent

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9769]
**Preferred term label:**
Temporal artery pulses reduced or absent

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Temporal arteritis and polymyalgia rheumatica: https://www.ncbi.nlm.nih.gov/books/NBK537274/ "Features like ... lack of pulses in the temporal artery...  are the red flags that should prompt an urgent evaluation for GCA.

"

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Should be a child of Absent pulse

**Comment by MickeySegal** [comment]
OK
